### PR TITLE
Pods Summary UI improvements

### PR DESCRIPTION
**Pull request by adfost**
A few improvements to pods UI:
1. There is now a search bar, making the Pods Summary like Volumes Summary.
2. Pod name field has been widened, in order to show the full pod names.
3. An unnecessary newline was removed from Volumes Summary.

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/TenantDetails/PodsSummary.tsx [hunk 1]
```diff
@@ -129,10 +134,30 @@ const PodsSummary = ({
         />
       )}
       <h1 className={classes.sectionTitle}>Pods</h1>
+      <Grid item xs={12} className={classes.actionsTray}>
+        <TextField
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/PodsSummary.tsx:138]: please use the existing `SearchBox` component.
```jsx
<SearchBox
            placeholder={"Search Users"}
            onChange={setFilter}
            overrideClass={classes.searchField} // use this to define custom width if required.
          />
```

File: portal-ui/src/screens/Console/Tenants/TenantDetails/PodsSummary.tsx [hunk 2]
```diff
@@ -129,10 +134,30 @@ const PodsSummary = ({
...
+          placeholder="Search Pods"
+          className={classes.searchField}
+          id="search-resource"
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/PodsSummary.tsx:141]: please avoid using id attribute. if it is intended for testing purposes,we could use data-test-id

as one of the [best practices](https://testcafe.io/documentation/402836/guides/basic-guides/best-practices#selector-strategy) recommended by TestCafe.

>Use custom attributes (like data-testid) whose sole purpose is to identify items with TestCafe. These attributes are unlikely to change during development, so you don’t need to rewrite your Selectors as frequently.

File: portal-ui/src/screens/Console/Tenants/TenantDetails/PodsSummary.tsx [hunk 3]
```diff
@@ -82,6 +83,10 @@ const PodsSummary = ({
     setDeleteOpen(true);
   };
 
+  const filteredRecords: IPodListElement[] = pods.filter((elementItem) =>
+    elementItem.name.includes(filter)
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/PodsSummary.tsx:87]: we could make it case insensitive filter if the values have mixed case.

**Comment by prakashsvmx** [comment]
@adfost Please resolve the conflicts.

**Review by prakashsvmx — DISMISSED** [review]
LGTM :+1:

File: portal-ui/src/screens/Console/Storage/StoragePVCs.tsx [hunk 4]
```diff
@@ -118,23 +118,12 @@ const StorageVolumes = ({
       )}
       <h1 className={classes.sectionTitle}>Persistent Volumes Claims</h1>
       <Grid item xs={12} className={classes.actionsTray}>
-        <TextField
+        <SearchBox
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Storage/StoragePVCs.tsx:121]: Please include the value prop as it is a new requirement for SearchBox
```suggestion
        <SearchBox
            value={filter}
```

File: portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx [hunk 5]
```diff
@@ -138,28 +139,14 @@ const TenantVolumes = ({
       )}
       <h1 className={classes.sectionTitle}>Volumes</h1>
       <Grid item xs={12} className={classes.actionsTray}>
-        <TextField
+        <SearchBox
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/VolumesSummary.tsx:142]: Same case:
```suggestion
        <SearchBox
            value={filter}
```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by cniackz — CHANGES_REQUESTED** [review]
Let's investigate why tests are failing prior the merge.
If they are failing due to this PR, then let's fix it.
Thanks!

**Comment by dvaldivia** [comment]
react code has warnings

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]
tested. lgtm.

**Comment by jinapurapu** [comment]
Screenshot from 2022-02-22 13-23-52
should this text be updated to "No matching pools found"?

**Review by bexsoft — APPROVED** [review]

**Review by jinapurapu — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]
Tested. LGTM

**Review by cniackz — APPROVED** [review]

**Pull request merged**